### PR TITLE
Some fixes in MigrationCreateCommand

### PR DESCRIPTION
**Pull request by Tobias4872**
Hi,

despite adding the missing import for class "EntityManager", I propose some changes to meet some more strict tslint rules.

Please let me know if this will work for you.

Tobias.

### CODE REVIEW

File: src/commands/MigrationCreateCommand.ts [hunk 1]
```diff
@@ -73,14 +73,16 @@ export class MigrationCreateCommand {
...
      */
     protected static getTemplate(name: string, timestamp: number): string {
-        return `import {MigrationInterface, QueryRunner, Connection} from "typeorm";
+        return `import {Connection, EntityManager, MigrationInterface, QueryRunner} from "typeorm";
 
 export class ${name}${timestamp} implements MigrationInterface {
 
-    async up(queryRunner: QueryRunner, connection: Connection, entityManager?: EntityManager): Promise<any> {
+    public async up(queryRunner: QueryRunner, connection: Connection, entityManager?: EntityManager): Promise<any> {
+        ;
```
Comment (pleerock) [inline src/commands/MigrationCreateCommand.ts]: this is really disgusting tslint rule.

Comment (Tobias4872) [inline src/commands/MigrationCreateCommand.ts]: We have not been setting this rule explicitly, it is part of the default rule set. In any case, would you still consider this pull request? Or do you need me to change anything?

Comment (pleerock) [inline src/commands/MigrationCreateCommand.ts]: yes, of course we need to have `EntityManager` in imports. I'm okay with `public` keyword (however don't really like it). But having `;` is not acceptable. Please change only this part.

**Comment by Tobias4872** [comment]
Done. Removed semicolons in empty functions.

**Comment by pleerock** [comment]
Thank you!

**Pull request merged**